Mathlib/CategoryTheory/Limits/Types.lean

```diff
@@ -31,6 +31,58 @@ universe v u
 
 namespace CategoryTheory.Limits.Types
 
+section limit_characterization
+
+variable {J : Type v} [Category J] {F : J ⥤ Type u}
+
+/-- Given a section of a functor F into `Type*`,
+  construct a cone over F with `PUnit` as the cone point. -/
+def cone_of_section {s} (hs : s ∈ F.sections) : Cone F where
```

Comment:
I'm not familiar with the new naming conventions at all, but should this be `coneOfSection`?

Reply:
Thanks, renamed